cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MatchTest.scala

```diff
@@ -178,21 +178,34 @@ class MatchTest extends DocumentingTest {
         p("Inside a single pattern, relationships will only be matched once. You can read more about this in <<cypherdoc-uniqueness>>.")
       }
       section("Relationship types with uncommon characters", "rel-types-with-uncommon-chars") {
-        val initQuery =
-          """MATCH (charlie:Person {name: 'Charlie Sheen'}), (rob:Person {name: 'Rob Reiner'})
-            |CREATE (rob)-[:`TYPE
-            |WITH SPACE`]->(charlie)"""
+
+        val isWindows = if (System.getProperty("os.name").toLowerCase.startsWith("windows")) true else false
```

Comment:
`val isWindows = System.getProperty("os.name").toLowerCase.startsWith("windows")`?

Reply:
Now that you mention it...